src/pages/MessagesAndFriends.tsx

```diff
@@ -1,16 +1,24 @@
+import { useState } from 'react'
 import { Box } from '@mui/material'
 import Header from 'components/header/Header'
 import TabsMessagesFriends from 'components/tabsMessagesFriends/TabsMessagesFriends'
 import UserProfile from 'components/userProfile/UserProfile'
+import { UserProfileData } from '../types/UserProfileData'
+import dataProfile from '../../src/components/userProfile/userProfile.json'
 
 const MessagesAndFriends = () => {
+  const [friendsData, setFriendsData] = useState(dataProfile)
```

Comment:
i would move logic with get/set data, click - to hook. Even now would do it, bc later with back end its possible to leave as it is